.travis.yml

```diff
@@ -36,3 +36,6 @@ env:
   - secure: wLNuKwwziqNyvZTpn8+uRJKkn8P8e0+CNLysCMA1ijtNYzCTlhnpzPvFiHEXmhGvFjhNuWkXXlVhmemzaFSZHxEyM/FLVXzKl782KQM9+uKx2jsvUemgqzA+SQyIqvjJYEMtg81UOXFkfnvAD1dtNdW8GcPW5/MREtcmSw9m7MfzeKigCKXnjiA772NA0fW2jiJaIHOMsp/7M+7IWX1a/U8F2Ta0BlvuZ/OYB66Ln4yfIVQ2HUZRGcuJzuQA+gbJzbnEAJqoAUAV7+AIY7qYG8igE1kYtYpJssnDQuMLMtrVmkaa4i6SelIMi1QjqyACNNC/3gsbtbYySPh+1o5JQwQLmI2hjo/A/v4a3lYFjTF4+4CtkHswUzBOgWR+g5lBPnxwpEc7sg4wuvJzw3dpZA3xJ0SotGyZQnNsXt+LbTKQlw1QVoho5CtpomxXX0D0y1aKl5JFR6wnBV5YovhWpdYOIQ2K9LbOIGX0YIvERXHS/lawYarnWimiIzvKFwRZy9xtlJtZYJoiJd/DqO6+CkyV+AujxPBgvEVZd/OfODIuWS+NRXUzeOxtIBSziZ73aqCHsO2GXH4VfsHy22xRzkGuAIpOIb+zpZAPnseiy+NJ4NHJemhvzE2VXncls4StUQWgaEir0GweaZPqL6QDRAN9gsPGxzZXRVyPXnYFXJ0=
   - secure: NCFc6zJrb8MpNhcnjkYYzQe0nk2ui0vgeBswkrxpOVLMEJxmcM53OabWLK0pW9VGdWMFMr9gndX8zOXyablMhTxUPsEbDZ4IZJr1MNdvFbx7qoohrKQ8nYYU6y73vuAf6p1NyrqOTAp2Ow9bI/boMLmhKvDKZ2dqlJDsJOF4e797wv8ZPXVpMgTcZE2n1zCA1MV3vWsJeoz5EgRrc0O/D/NmOBnQYrn5NiT3h33wFvApVEv/z/KYBWu15mx9mePw6OhEFv/+dcOHw1t6DK4WwtC5kZuDE+yTEz4il5R9QsG2J2uXenXRfACYcwVbRTlXSyELnDpmbrJDCrWdrv0k4pSxdUjZJkqXORK6kVkjq03tD85bYv76S0cI1D73ezn5tj/4PSRDL6j47hnQtGsiVGnJ0XuwAuGi9eXiL84ROblkyjUgF2ZniEYQdg0Kg5WDJXIRD3D7gL3CkQ4bfLFzgl1HQLMYPFmHrWVt7rMzWjFxKHFt6cS7HSTJVr5nFudPvQ4Ozzl56asMnV5eNqksGdSioKeL65x1S2VihKeZOViHH9vIIOU39kJNMlFRf1j+7zewboFOGkA9UKnt9aK9n6fjFm5BrpDwRZoqOfq1raKwux/cxs774ZGE3AfVVvH6rw/xxdQjj0m76Qp+nLlJXAbVbiXD6+lx/22u2d7D9T0=
   - secure: pVE+P4F7b14BhGBUXYhPiTBYwsWj50jOQ76Lwae5IcWFKB8Ibrr8bvIW/bLfnsjVv51d0Bf0vyGRHoW2V2POC6n1wkzE3nHZ+lSJQkmg5jsqr9z4Nxd6EsZUpwlSBC7/hzlqbE8GeBeIg5qxoclInZrcG+zoTgcfA1N/hIooNniP3TE1jBkRueYptvAiG8ugL9IwbQ8UvXXE9GVHkROwax4AOEFG7ZjWDzOH2pzvGtvT34pPDGtL0ZYWwbklZsdkCkFmHR0Lo8vQi1sesqi3KmiqStlrRfEHw+RgRhk2tgB76AzIyUO4zOX0+nGqKWwfTGt933/19ENrg+uKJr1j57N9Cwzh2xxRGlwUi44c64lmOCMtWedU+xAF8omVvoOuqYibDTeNPm8lYaiV0JWoxrJ9vKwh3jEyi9lbUmYpU7I+Pi5SC2crQQQDYH/Qfd+gKY/V7QfCKfjjneKe/TA0uXBrHTv07E+WLGcLFVQ6ir3cvzk1lU5iyH2cR1mfCMY5/DN7EQQOJn5Ww1dwyJBPvJkBGHV/hR3QZUWtcE7nuvmbrFMZQBLzlnYuZfMtdRuyU8dOUtsDrt0On1A1JswvOfwAqSyitQ7mAUlq2mqazMRTNFmj8zVPZ12IWuMsEw4zTc16RAedszME166B2fp9PEMF4oErGlHxoBqhrpanyO0=
+  matrix:
```

Comment:
This change probably not wanted in master branch.

Reply:
Yeah, I agree TBH.